### DISCUSSION TITLE
Y axis input shaping, how bad is this?

### DISCUSSION THREAD

**Discussion by dasb00ter** [bassamanator/Sovol-SV06-firmware#175] [General]
Is this too much wobble in my hardware(I could have swore I had everything tight) or is this typical of a big bed slinger. What do others y axis's look like for the sv06 plus(big bed slinger). I have no frame of reference.

My accelerometer for reference:
![IMG_20240725_161322](https://github.com/user-attachments/assets/4336f07a-a724-4390-892f-5b8e1fcf93ed)

accelerometer clamped to front of bed in this orientantion gleans the following graph:
![IMG_20240726_094314](https://github.com/user-attachments/assets/3d7f5021-d1ec-4f81-a7f2-9228cdf122b0)
![shaper_calibrate_y](https://github.com/user-attachments/assets/bf18852b-211a-48b4-950a-42f1d1f96040)

Accelerometer screwed into tab on pei sheet in this orientation and taped to bed
![IMG_20240726_094340](https://github.com/user-attachments/assets/ca8eb76a-2d87-4d88-a6b8-17acfc55cbca)
![shaper_calibrate_ytape](https://github.com/user-attachments/assets/41cb3fb4-72ef-44d2-bcc5-348b71fe9b18)
I'm also a little confused why the Z axis seems the one that is active shouldn't it be X

**Comment by Ederhex** [comment]
Ok so sometimes it's hard to tell the exact reason but there are some hints in the graphs.

The X + Y resonances might be unaligned . You can tell by looking at the X and Y graphs. They peak at different frequencies and when they peak you see that the vibrations on the other axis are far less meaning one is resonating. Most of the time this is due belt tensions that are too different from each other (and they will resonance at a different frequency). Bad belt tension can cause multiple spikes while we aim for 1.

When I tension my belts I use the technique described here https://old.reddit.com/r/Sovol/comments/189hctc/using_pitchsound_frequency_to_get_correct_belt but a simple belt tuner tool like https://www.printables.com/model/135075-belt-tension-gauge also works great (aiming for middle point for both belts).

Other factors that can badly influence input shaper:

- Broken bearings (or bearings needing maintenance)
- Loose screws/bolts/parts
- Bearings on the top Z axis
- Wobbly table

Also the accelerometer has an axis map on the chip so sometimes the data is interpreted wrong when you flip the accelerometer. You can see some clues for the axis on the chip (arrows with axis). I think the last graph your Z is actually your Y or X axis.

I hope this is any help to you 

Good luck

**Comment by dasb00ter** [comment]
So part of the problem was at least my "learning". I had followed a video and remapped the axis in my config during testing. Then I revisited a month later the process and forgot the remap. 

Belts were too loose that is definitely a problem. Still although the X axis was easy to lock down and tune the Y remained ....odd. Using several frequency tuning apps the Y axis jumped between two frequencies? I finally took my guitar and listened to what the frequency should be tuned to. I used the sound and made the belt comparable then I tuned to the higher of the 2 numbers that it was jumping back and forth from. 

I think my graph improved but I think there is still a prominent secondary hump in the Y. 

Thanks for the suggestion
